generators/flutter-mobile/templates/base/.gitignore

```diff
@@ -0,0 +1,46 @@
+# Miscellaneous
+*.class
+*.log
+*.pyc
+*.swp
+.DS_Store
+.atom/
+.buildlog/
+.history
+.svn/
+
+# IntelliJ related
+*.iml
+*.ipr
+*.iws
+.idea/
```

Comment:
I'm not fond of having IDE files in the project gitignore since they should go in everyone's global gitignore.

Reply:
This is the default Flutter project's `gitignore`. Won't it unsettle users to remove those usual rules ?

Reply:
Not sure, to me it's a best practice to not putt editor files in the project gitignore or else it will become polluted with every dev own custom editor.

But if it's the default flutter projet gitignore why not keeping it as it :shrug: